themes/src/main/resources-community/theme/base/account/messages/messages_ar.properties

```diff
@@ -0,0 +1,352 @@
+readOnlyPasswordMessage=لا يمكنك تحديث كلمة مرورك لأن حسابك للقراءة فقط
+
+successTotpMessage=مصادقة المحمول تكوينها.
+successTotpRemovedMessage=تمت إزالة مصادقة المحمول
+
+successGrantRevokedMessage=تمت إزالة المنحة بنجاح
+
+accountUpdatedMessage=تم تحديث الحساب الخاص بك
+accountPasswordUpdatedMessage=لقد تم تحديث كلمة السر الخاصة بك
+
+missingIdentityProviderMessage=موفر الهوية غير محدد
+invalidFederatedIdentityActionMessage=إجراء غير صالح أو مفقود
+identityProviderNotFoundMessage=موفر الهوية المحدد غير موجود
+federatedIdentityLinkNotActiveMessage=هذه الهوية لم تعد نشطة
+federatedIdentityRemovingLastProviderMessage=لا يمكنك إزالة آخر هوية متحد لأنه ليس لديك كلمة مرور
+identityProviderRedirectErrorMessage=فشل في إعادة التوجيه إلى موفر الهوية
+identityProviderRemovedMessage=تمت إزالة مزود الهوية بنجاح
+identityProviderAlreadyLinkedMessage=الهوية المتحدة التي تم إرجاعها بواسطة {0} مرتبطة بالفعل بمستخدم آخر
+staleCodeAccountMessage=انتهت صلاحية الصفحة. حاول مرة اخرى.
+consentDenied=رفض الموافقة.
+
+accountDisabledMessage=الحساب معطل ، اتصل بالمسؤول
+accountTemporarilyDisabledMessage=الحساب معطل مؤقتًا ، أو اتصل بالمسؤول ، أو أعد المحاولة لاحقًا.
+invalidPasswordMinLengthMessage=كلمة مرور غير صالحة: الحد الأدنى للطول {0}
+invalidPasswordMinLowerCaseCharsMessage=كلمة مرور غير صالحة: يجب أن تحتوي على {0} أحرف صغيرة على الأقل.
+invalidPasswordMinDigitsMessage=كلمة مرور غير صالحة: يجب أن تحتوي على {0} رقم رقمي على الأقل.
+invalidPasswordMinUpperCaseCharsMessage=كلمة مرور غير صالحة: يجب أن تحتوي على {0} أحرف كبيرة على الأقل.
+invalidPasswordMinSpecialCharsMessage=كلمة مرور غير صالحة: يجب أن تحتوي على {0} أحرف خاصة على الأقل.
+invalidPasswordNotUsernameMessage=كلمة مرور غير صالحة: يجب ألا تكون مساوية لاسم المستخدم.
+invalidPasswordRegexPatternMessage=كلمة مرور غير صالحة: فشلت في مطابقة أنماط regex
+invalidPasswordHistoryMessage=كلمة مرور غير صالحة: يجب ألا تكون مساوية لأي من كلمات المرور {0} الأخيرة.
+invalidPasswordBlacklistedMessage=كلمة مرور غير صالحة: تم إدراج كلمة المرور في القائمة السوداء.
+invalidPasswordGenericMessage=كلمة المرور غير صالحة: كلمة المرور الجديدة لا تتوافق مع سياسات كلمة المرور الخاصة بنا.
+
+# Authorization
+myResources=الموارد الخاصة بي
+myResourcesSub=الموارد الخاصة بي
+doDeny=أنكر
+doRevoke=انسحاب
+doApprove=يوافق
+doRemoveSharing=رحيل مشاركة
+doRemoveRequest=أزل الطلب
+peopleAccessResource=الأشخاص الذين لديهم حق الوصول إلى هذا المورد
+resourceManagedPolicies=أذونات منح الوصول إلى هذا المورد
+resourceNoPermissionsGrantingAccess=لا توجد أذونات تمنح حق الوصول إلى هذا المورد
+anyAction=اي فعل
+description=وصف
+name=وصف
+scopes=نطاقات
+resource=نطاقات
+user=المستعمل
+peopleSharingThisResource=الأشخاص الذين يشاركون هذا المورد
+shareWithOthers=شارك مع الآخرين
+needMyApproval=بحاجة الى موافقتي
+requestsWaitingApproval=طلباتك في انتظار الموافقة
+icon=أيقونة
+requestor=الطالب
+owner=صاحب
+resourcesSharedWithMe=الموارد المشتركة معي
+permissionRequestion=طلبات إذن
+permission=الإذن
+shares=شارك
+
+locale_ca=Catal\u00e0
```

Comment:
This is mixing encoding. The locale_* properties doesn't need to be added here as they can be inherited from messages_en.properties instead.

Reply:
Thank you